src/main/java/code401challenges/InsertionSort.java

```diff
@@ -0,0 +1,24 @@
+package code401challenges;
+
+public class InsertionSort {

```

Comment:
I recommend making a package called "sort" where you can put your sorting algorithms.